assets/scss/_variables_project.scss

```diff
@@ -23,68 +23,84 @@ $navbar-dark-disabled-color: rgba($black, 0.25);
 
 .lead p {
   font-weight: 400;
-  color: $secondary; }
+  color: $secondary;
+}
+  
+p {
+  margin-left: 1em;
+  margin-right: 1em;
+}
 
 // Front page banner
 
 .banner {
   margin: 3em 1em;
   text-align: center;
-}
-  .banner > .banner-image {
+
+  .banner-image {
     margin-bottom: 2em; 
     margin-left: auto;
     margin-right: auto;
     max-width: 65%;
   }
-  .banner > .banner-text {
+
+  .banner-text {
     margin-bottom: auto;
     margin-top: auto;
   }
-  .banner > .banner-text h1 {
+
+  .banner-text h1 {
     font-size: 2em;
   }
-  .banner > .banner-text p {
+
+  .banner-text p {
     font-family: inherit;
     font-size: 1.5em;
     font-weight: 500;
     line-height: 1.2;
+    margin-left: 0;
+    margin-right: 0;
   }
+}
 
 .banner-image {
   display: block;
   max-width: 55%;
 }
-  .banner-image img {
-    display: block;
-    height: auto; 
-    width: 100%;
-  }
+
+.banner-image img {
+  display: block;
+  height: auto; 
+  width: 100%;
+}
 
 @media screen and (min-width: 480px) {
   .banner {
     text-align: center;
-  }
-    .banner > .banner-image {
+    .banner-image {
       margin-left: auto;
       margin-right: auto;
     }
   }
+}
 
 @media screen and (min-width: 769px) {
-  .banner {
-    display: flex;
-  }
-    .banner > .banner-image {
-      display: block;
-      max-width: 100%;
-    }
-    .banner > .banner-image > img {
-      margin: 0 auto;
-      max-width: 80%;
-    }
-    .banner > .banner-text {
-      flex-basis: 60%;
-      text-align: left;
-    }
-  }
+	.banner {
```

Comment:
Leading whitespace in these next few blocks looks a bit off here compared to the rest of the file, is this intentional?

Reply:
Hmmm, that's very strange, I was trying out using another text editor to update the SCSS, so it must be something to do with that.

I fixed this in VI... thanks for spotting!